test/test_autograd.py

```diff
@@ -2690,14 +2690,10 @@ def maybe_non_contig(tensor):
             if isinstance(arg.tensor, Variable):
                 return maybe_non_contig(arg.tensor)
             return Variable(maybe_non_contig(arg.tensor))
-        elif torch.is_tensor(arg) or isinstance(arg, Variable):
-            # TODO: simplify once Variable and Tensor are merged
-            shape = arg.shape
-            if not torch.is_tensor(arg):
-                arg = arg.data
+        elif isinstance(arg, torch.Tensor):
```

Comment:
can we now deprecate `torch.is_tensor`?

Reply:
We still have a bunch of internal calls to `torch.is_tensor` but we should be able to deprecate it before release